test/python/gemm/frost/test_public_execute_flavors.py

```
@@ -0,0 +1,254 @@
+    g = cudnn.pygraph(io_data_type=BF16, intermediate_data_type=F32, compute_data_type=F32)
+    A = g.tensor(name="A", uid=1, dim=[1, M, K], stride=[M * K, K, 1])
+    B = g.tensor(name="B", uid=2, dim=[1, K, N], stride=[K * N, 1, K])
+    C = g.matmul(A=A, B=B, name="mm")
+    C.set_output(True).set_data_type(BF16).set_uid(3)
+    _pin_frost(g)
+
+    a, b, _ = _operands()
+    c = torch.empty(1, M, N, dtype=torch.bfloat16, device="cuda")
+    pack = g._normalize(g._uid_to_data({1: a, 2: b, 3: c}), tiny)
+    assert pack.workspace_bytes == 16
+    with pytest.raises(ValueError, match="needs a .*-byte workspace"):
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)